deploy_hpo.sh

```diff
@@ -33,19 +33,24 @@ non_interactive=0
 hpo_ns=""
 # docker: loop timeout is turned off by default
 timeout=-1
+service_type="both"
 
 # source the helpers script
 . ${SCRIPTS_DIR}/cluster-helpers.sh
 . ${SCRIPTS_DIR}/openshift-helpers.sh
 
 function usage() {
 	echo
-	echo "Usage: $0 [-a] [-c [docker|minikube|native|openshift]] [-o hpo container image] [-n namespace] [-d configmaps-dir ]"
-	echo " -s = start(default), -t = terminate"
-	echo " -c: cluster type."
-	echo " -o: build with specific hpo container image name [Default - kruize/hpo:<version>]"
-	echo " -n: Namespace to which hpo is deployed [Default - monitoring namespace for cluster type minikube]"
-	echo " -d: Config maps directory [Default - manifests/configmaps]"
+	echo "Usage:"
+	echo " -a | --non_interactive	: interactive (default)"
+	echo " -s | --start				: start(default) the app"
+	echo " -t | --terminate			: terminate the app"
+	echo " -c | --cluster_type		: cluster type [docker|minikube|native|openshift]]"
+	echo " -o | --container_image	: build with specific hpo container image name [Default - kruize/hpo:<version>]"
+	echo " -n | --namespace			: Namespace to which hpo is deployed [Default - monitoring namespace for cluster type minikube]"
+	echo " -d | --configmaps		: Config maps directory [Default - manifests/configmaps]"
+	echo " -b | --both				: install both REST and the gRPC service"
+	echo " -r | --rest				: install REST only"
```

Comment:
Can you align the options so that they show up consistently on the terminal

Reply:
Can we remove the `-b` and `-r` options and only have `--both` and `--rest` options
Change `--configmaps` option to `--configmaps_dir`

Reply:
done